composer.json

```diff
@@ -11,6 +11,7 @@
         "psr/log": "^1.0",
         "league/plates": "^3.1",
         "paragonie/random_compat": "^1.2",
-        "symfony/polyfill-php56": "^1.1"
+        "symfony/polyfill-php56": "^1.1",
+        "symfony/polyfill-php55": "^1.1"
```

Comment:
That's a polyfill that provides PHP 5.5 features to older versions. Including `password_verify` :)

https://github.com/symfony/polyfill